app/views/people/show.html.haml

```diff
@@ -12,6 +12,7 @@
           %p
             = t('.mp_for')
             = link_to(@current_incumbency.constituency.name, constituency_path(@current_incumbency.constituency.graph_id))
+        - ViewHelper.convert_house_membership_status(@person.statuses[:house_membership_status])
```

Comment:
If you include the ViewHelper inside application controller, can you drop the ViewHelper call here?